src/widgets/capturelauncher.cpp

```diff
@@ -69,10 +71,26 @@ CaptureLauncher::CaptureLauncher(QDialog* parent)
             static_cast<void (QSpinBox::*)(int)>(&QSpinBox::valueChanged),
             this,
             [this](int val) {
-                QString sufix = val == 1 ? tr(" second") : tr(" seconds");
-                this->m_delaySpinBox->setSuffix(sufix);
+                QString suffix = val == 1 ? tr(" second") : tr(" seconds");
+                this->m_delaySpinBox->setSuffix(suffix);
             });
 
+    m_width = new QLineEdit("0");
+    m_height = new QLineEdit("0");
+    m_xOffset = new QLineEdit("0");
+    m_yOffset = new QLineEdit("0");
+
+    setLastRegion();
+
+    auto* dims_layout = new QHBoxLayout();
```

Comment:
It's not really my place to tell you this, but can you keep the naming style camelCase? I have a particular allergy to inconsistency.

Reply:
Yeah sorry, i have been doing like 70% rust coding and am in the headspace of their conventions.